Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/heavy_rifle_nt.yml

```
@@ -0,0 +1,99 @@
+    layers:
+    - state: base
+      map: ["enum.AmmoVisualLayers.Base"]
+  - type: Appearance
+  - type: SpentAmmoVisuals
+  - type: StaticPrice
+    price: 10
+
+- type: entity
+  id: CartridgeHeavyRifleNT
+  name: cartridge (7,62mm NT)
+  parent: [ BaseCartridgeHeavyRifleNT, BaseRestrictedContraband ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)